CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+## [1.0.0] - 2021-07-08
+
+### Added
+- The `vault` UpstreamAuthority plugin now supports Kubernetes service account authentication (#2356)
+- A new `cert-manager` UpstreamAuthority plugin is now available (#2274)
+- SPIRE Server CLI can now be used to ban agents (#2374)
+- SPIRE Server CLI now has `count` subcommands for agents, entries, and bundles (#2128)
+- SPIRE Server can now be configured for SPIFFE federation using the `https_web` and `https_spiffe` profiles (#2340)
+- SPIRE Server and Agent now expose the standard gRPC health service (#2057, #2058)
+- SPIFFE bundle endpoint URL is now configurable (#2278)
+- SPIRE Agent may now optionally provided unregistered callers with a bundle for SVID validation via the `allow_unauthenticated_verifiers` configurable (#2102)
+- SPIRE Server JWT key type is now independently configurable via `jwt_key_type` (#1991)
+
+### Changed
+- SPIRE Server's SVID now uses the key type configured as `ca_key_type` (#2269)
+- Caller address is now logged for agent API calls resulting in an error (#2281)
+- Agent SVID renewals are now logged by the server at the INFO level (#2309)
+- Workload API JWT-SVID profile will now return an error if the caller is unidentified (#2369)
+- Workload API JWT-SVID profile will no longer return non-SPIFFE claims on validated JWTs from foreign trust domains (#2372)
+- The SPIFFE bundle endpoint TLS stack now supports a smaller set of signature schemes (#2397)
```

Comment:
It would be good if we can provide some more detail here. Any suggestions @azdagron?

Reply:
Maybe something like `The SPIFFE bundle endpoint supports a reduced signature algorithm set to improve key manager compatibility`?

Reply:
Oops sorry I meant more detail as in, what exactly is supported or not supported now (as opposed to why). I checked the PR and saw three signature schemes but they were a mouthful :) perhaps there's a different way to say the equivalent thing?